src/ploomber/telemetry/telemetry.py

```diff
@@ -57,6 +55,35 @@
 PLOOMBER_HOME_DIR = os.getenv("PLOOMBER_HOME_DIR")
 
 
+class UserSettings(Config):
+    """User-customizable settings
+    """
+    version_check_enabled: bool = True
+    cloud_key: str = None
+    stats_enabled: bool = True
+
+    @classmethod
+    def path(cls):
+        return Path(check_dir_exist(CONF_DIR), DEFAULT_USER_CONF)
+
+
+class Internal(Config):
```

Comment:
Maybe call it ploomber settings?

Reply:
I want to be more explicit that these are for internal use and others are for users.